emacs/merlin-imenu.el

```diff
@@ -1,10 +1,12 @@
 ;;; merlin-imenu.el --- Merlin and imenu integration.   -*- coding: utf-8 -*-
 ;; Licensed under the MIT license.
 
-;; Author: tddsg
-;; Created: 10 July 2016
-;; Updated: 27 April 2017
-;; Version: 0.2
+;; Author: tddsg (Ta Quang Trung)
```

Comment:
Ah, I wasn't aware you were the initial author. Feel free to erase the username and just keep your real name if you like.

Reply:
Thanks! But actually I am OK with your suggestion, so I think that I can keep the current information as it is.